test/routes/issueRoutesTest.js

```diff
@@ -0,0 +1,317 @@
+const chai = require('chai');
+const expect = require('chai').expect;
+const mongoose = require('mongoose');
+
+chai.use(require('chai-http'));
+
+const Issue = require('../../models/issueModel');
+const path = require('path');
+const File = require(path.resolve('./models/fileModel'));
+
+const testRoutes = require('../../routes/issueRoutes');
+
+describe('Issue requests', function(){
+
+    const request = chai.request;
+
+    describe('Create issue', function(){        
+
+        describe('Valid params', function() {
+
+            let issue;
+            before((done) => {
+                issue = {
+                    title: "Route issue",
+                    description: "Create route issue test",
+                    name: "Blah"
+                };
+                done();
+            });
+            
+            it('should save the issue', function(done) {
+                request('http://localhost:8000')
+                .post('/issues')
+                .send(issue)
+                .then((response) => {
+                    
+                    const createdIssue = response.body;
+
+                    expect(createdIssue).to.be.an('object');
```

Comment:
Make sure you always check response code (like you did on some other examples). This will help you identify reason for test failure a lot quicker.

For example: If your test fails and you see that status code is 404. You will right away know that resources hasn't been found and will have general idea where to look for bug. If you miss the status code you would get failure saying (in your test case) 'expected createdIssue to be an 'object' but found null'. This only says that issue is not there - you don't know the real reason for it. It could be bad request (400), internal server error (500), unauthorized (401), etc.